Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -606,7 +606,21 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
             }
 
             // callbacks to update the content
-            marker.refreshContent(entry: e, highlight: highlight)
+            if let marker = marker as? IMultiValueMarker {
+                var markerData: [MarkerData] = []
+                var dataSetIndex = 0
+                for dataSet in data?.dataSets ?? [] {
```

Comment:
What do you think of using this instead of the index variable/increment:

`for (dataSetIndex, dataSet) in (data?.dataSets ?? []).enumerated()`
or
`(data?.dataSets ?? []).enumerated().forEach`